File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:

        if len(s) != len(t):
            return False

        remaining = s

        for l in t:
            if s.count(l) == t.count(l):
                s.replace(l, "")
            else:
                return False
        
        return True
        

File: Data Structures & Algorithms/is-palindrome/submission-1.cs
public class Solution {
    public bool IsPalindrome(string s) {

        string str = Regex.Replace(s, "[^a-zA-Z0-9]", String.Empty).ToLower();

        Console.WriteLine(str);

        int left = 0;
        int right = str.Length - 1;

        while (left < right) {
            if (str[left] != str[right]) {
                return false;
            }
            left++;
            right--;
        }
        return true;
        
    }
}
